web/client/epics/geostory.js

```diff
@@ -8,12 +8,16 @@
  */
 
 import { Observable } from 'rxjs';
-import head from 'lodash/head';
-
-import isNaN from 'lodash/isNaN';
-import isString from 'lodash/isString';
-import isNil from 'lodash/isNil';
-import lastIndexOf from 'lodash/lastIndexOf';
+import {
+    head,
+    isNaN,
+    isString,
+    isNil,
+    lastIndexOf,
+    words,
+    replace,
+    get
+} from 'lodash';
```

Comment:
We are trying to update new files (when possible) using the lodash module (eg. `lodash/iNaN`) instead the whole import of the library so could you please keep this kind of import for the files you edited in this PR?